docs/source/resources/troubleshooting.md

```diff
@@ -30,9 +32,10 @@ issue](https://github.com/NVIDIA/NVTabular/issues/429).
 
 NVTabular is designed to scale to larger than GPU or host memory datasets. In our experiments, we are able to [scale to 1.3TB of uncompressed click logs](https://github.com/NVIDIA/NVTabular/tree/main/examples/scaling-criteo). However, some workflows can result in OOM errors `cudaErrorMemoryAllocation out of memory`, which can be addressed by small configuration changes.
 
-### 1. Setting the Row Group Size for the Parquet Files
+### Setting the Row Group Size for the Parquet Files
```

Comment:
Numbering doesn't make sense to me:

A) The sections are listed sequentially, so browsing readers should encounter the sections in order.
B) Readers brought to this page by google will likely Ctrl+F for the search term and sequence won't matter.

Reply:
ok sounds good